tests/build/build.go

```diff
@@ -507,16 +507,25 @@ var _ = framework.BuildSuiteDescribe("Build service E2E tests", Label("build", "
 
 				for i := range gatherResult {
 					if gatherResult[i] == "sanity-inspect-image" {
+						// Fetching HACBS_TEST_OUTPUT shouldn't fail
```

Comment:
This comment IMO shouldn't be here. sanity-inspect-image doesn't return `HACBS_TEST_OUTPUT`

Reply:
In Below Line 511 its checking for BASE_IMAGE generation not the HACBS_TEST_OUTPUT

Reply:
but comment says `HACBS_TEST_OUTPUT`

Reply:
done. Lets see if CI passes now.